lib/commands.js

```diff
@@ -34,6 +39,50 @@ function noPermissions (message) {
   }).catch(() => message.channel.send(errorMessage + ' here'))
 }
 
+function searchMainGithubIssues (value) {
+  return fetch(`https://api.github.com/search/issues?q=repo:runelite/runelite+${value}`, {
+    headers: {
+      Authorization: `token ${config.githubToken}`
+    }
+  }).then(res => res.json())
+}
+
+function addCommentToGithubIssue (id, message) {
+  searchMainGithubIssues(id).then(body => {
+    if (body.items.length === 0) {
+      return
+    }
+
+    const commentsURL = body.items[0].comments_url
+
+    if (!commentsURL) {
+      return
+    }
+
+    return fetch(commentsURL, {
+      method: 'POST',
+      headers: {
+        'Authorization': `token ${config.githubToken}`,
+        'User-Agent': 'RuneLite-Bot',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        body: message
+      })
+    }).catch(e => log.debug(e))
+  })
+}
+
+function addCommandRan(command) {
+  commandsRan.push(command);
+  setTimeout(() => {
```

Comment:
I don't know enough about JS async. Is it possible for two setTimeout callsbacks to trigger in such a manner that by the time you get to line 81, the value of `i` is incorrect? Or is there a guarantee that this function once entered will get to completion before another instance of the callback is ran?